panda-qa/Resources/CommonFunctionality.robot

```diff
@@ -191,13 +189,13 @@ Start FN Article without byline
     Maximize Browser Window
 
 Start Mansion Global Article without byline
-    Set Selenium Speed  1 seconds
+    Set Selenium Speed  0.5 seconds
     ${options} =  Set Browser Options
     IF  "${Env}" == "prod"
-        Open Browser  https://www.mansionglobal.com  ${Browser}  options=${options}
+        Open Browser  https://www.mansionglobal.com  ${Browser}
```

Comment:
do not remove   options=${options} at the end. please update it everywhere you changed.